src/assigner.php

```diff
@@ -25,6 +48,8 @@
             // if groups have been successfully updated,
             // invalidate the cache
             Auth::invalidateUserGroupCache();
+            // Update the last usage time
+            $_SESSION['lastAssignerUsage'] = $currentTime;
```

Comment:
This would be best handled in Assigner class. Also, this allows a user to spam the assigner using multiple sessions. It would be best to use cache to save last assigner usage by TS CLDBID/UID, similarly to login code cache.